06-Customizing AzTS for your org/Extending AzTS/FeaturesInCMET.md

```diff
@@ -0,0 +1,91 @@
+## Features in CMET
+- ### Bulk Edit
+
+With Bulk Edit feature, custom tags can be added to more than 1 controls at a time.
+
+### **Enabling Bulk Edit feature in CMET**
+
+To enable bulk edit in CMET (Control Metadata Editor Tool), you need to configure following two application settings in AzTS API App service:
+
+1.	FeatureManagement__BulkEdit -> True
+2.	UIConfigurations__ControlEditorFeatureConfiguration__IsBulkEditEnabled -> True
+
+Please follow below steps to add these settings.
+1. Open the Azure portal.
+2. Navigate to AzTS host subscription -> AzTS host resource group.
+3. Go to app service AzSK-AzTS-WebApi-xxxxx.
+4. Go to Configuration under Settings.
+5. Click on "New application setting" to add required application settings.
+
+![Resources](../../Images/06_BulkEdit_1.png)
+
+![Resources](../../Images/06_BulkEdit_2.png)
+
+
+### **Using Bulk Edit feature to add Custom tags to controls** 
+
+Once Bulk Edit Feature is enabled, follow below steps to add custom tag(s) for set of controls.
+
+1. Open AzTS UI in new browser window.
+2. Open CMET editor (for this user will require editor permission over CMET, if you don’t have permission, please follow steps mentioned [here](../Extending%20AzTS/Prerequisites.md#access-to-cmet-control-metadata-editor-tool)).
```

Comment:
(To open the CMET editor, you need to have editor permissions. If you don't have the required permission, please follow the steps mentioned here)

Reply:
Updated